assets/js/contact_form.js

```diff
@@ -0,0 +1,8 @@
+function sendForm(){
+    document.getElementById("form-redirect").value = "http://"+window.location.host;
```

Comment:
It should be `https`

Reply:
If i use https doesn't redirect in localhost correctly, but in this case using http at production it will be redirected to https